backend/api/views/main.py

```diff
@@ -24,26 +24,34 @@ def get_persons():
     return create_response(data={"persons": persons})
 
 
-# function that is called when you visit /census_response
+
+# Given:
+# date in database
+# (optional) tract_id in database
+#
+# Returns all response rates associated to that date (and id)
 @main.route("/census_response", methods=["GET"])
 def get_census_response():
-    responses = CensusResponse.objects()  # CURRENTLY DOESNT HAVE DATE BUT USE IT LATER
+    responses = CensusResponse.objects()
     response_rates = []
-    has_date = "date" in request.args
-
-    if has_date:
-        date = request.args["date"]
-        year = date[-4:]
-    else:
-        year = request.args["year"]
+    check_tract_id = False
+    tract_id = 0
+    date = request.args["date"]
+    if "tract_id" in request.args:
+        tract_id = request.args["tract_id"]
+        check_tract_id = True
+    DATE_INDEX = -4
```

Comment:
this is great to have! it'd be better if you could put it near the top of the file (or if we do have enough constants, we'll take those out into another file)